content/blog/entries/cc-vocabulary-docs-updates-2/contents.lr

```diff
@@ -0,0 +1,87 @@
+title: Vocabulary Site Mid-Internship Update (v2)
+---
+categories:
+cc-vocabulary
+gsod
+gsod-2020
+---
+author: nimishbongale
+---
+series: gsod-2020-vocabulary-usage-guide
+---
+pub_date: 2020-11-09
+---
+body:
+This is a mid-internship blog post. Wait. what!? Already? Let's glance over my progress, shall we?
+
+## Vocabulary Site Updates (Edition 2/many more to come)
+
+Oh boy! 1.5 months have passed since I've been investing time in building a landing site & usage guide for `CC Vocabulary`. A lot has changed since the time of posting my last blog post. **A lot**.
```

Comment:
Not a code snippet, so no need to put in backticks.
```suggestion
Oh boy! 1.5 months have passed since I've been investing time in building a landing site & usage guide for CC Vocabulary. A lot has changed since the time of posting my last blog post. **A lot**.
```